### DISCUSSION TITLE
How to make WiFi connections in Uno+GTK

### DISCUSSION THREAD

**Discussion by LionelXN** [unoplatform/uno#14174] [Q&A] [answered]
Is there a recommended method? The functions to be implemented include: obtaining WiFi list, connecting, disconnecting, and deleting

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! The WinRT API for this is `Windows.Devices.WiFi.WiFiAdapter` and while it is present in Uno, it is not implemented at this time.

That being said, you can use platform specific API calls for Windows, Linux and mac to get that information.

**Reply by LionelXN** [reply]
Thank you for your reply
